### ISSUE TITLE
knife status NoMethodError: undefined method `first' for nil:NilClass

### ISSUE THREAD

**Issue by danielcbright** [chef/chef#10774]
## Description
When running `knife status  "name:*node*" -F json` the expected result is to show the status of the node searched for, instead there is an error given.

## Chef Version
Chef Infra Client `14.13.11`
Chef Workstation Version `latest stable`

## Platform Version
Any Flavor Linux/Windows EC2 Instance

## Replication Case
Run the `knife status "name:*node*"` command against any EC2 instance that **_doesn't have a public IP_**.

## Client Output
```
/opt/chef-workstation/embedded/lib/ruby/2.7.0/net/http/exceptions.rb:25: warning: already initialized constant Net::HTTPClientException
/opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.6.14/lib/chef/monkey_patches/net_http.rb:16: warning: previous definition of HTTPClientException was here
ERROR: knife encountered an unexpected error
This may be a bug in the 'status' knife command or plugin
```

## Stacktrace
```
/opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife/core/status_presenter.rb:70:in `block in summarize_json': undefined method `first' for nil:NilClass (NoMethodError)
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife/core/status_presenter.rb:65:in `each'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife/core/status_presenter.rb:65:in `summarize_json'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife/core/status_presenter.rb:57:in `format'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife/core/ui.rb:184:in `output'
	from /opt/chef-workstation/embedded/lib/ruby/2.7.0/forwardable.rb:235:in `output'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife/status.rb:87:in `run'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife.rb:490:in `block in run_with_pretty_exceptions'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/local_mode.rb:42:in `with_server_connectivity'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife.rb:489:in `run_with_pretty_exceptions'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/knife.rb:229:in `run'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/lib/chef/application/knife.rb:165:in `run'
	from /opt/chef-workstation/embedded/lib/ruby/gems/2.7.0/gems/chef-16.5.77/bin/knife:24:in `<top (required)>'
	from /opt/chef-workstation/bin/knife:353:in `load'
	from /opt/chef-workstation/bin/knife:353:in `<main>'
```

**Comment by amcappelli** [comment]
I was able to reproduce this with version 16.8.14 of chef-client as well

**Comment by phiggins** [comment]
Fixed in #10781.
